server/app/uk/gov/ons/addressIndex/server/controllers/AddressIndexController.scala

```diff
@@ -1,5 +1,7 @@
 package uk.gov.ons.addressIndex.server.controllers
 
-import play.api.mvc.Controller
+import uk.gov.ons.addressIndex.server.modules.AddressIndexCannedResponse
 
-abstract class AddressIndexController extends Controller
\ No newline at end of file
+abstract class AddressIndexController
```

Comment:
Basically, adding this on abstract layer means all controllers have the canned responses. The self cake pattern means the actions will use canned responses. This means that you can refer to each abstraction generically. If you want to modify one canned response then you can override it in the impl controller without modifying any code anywhere else :).